src/main/java/com/example/event_manager/service/EventServiceImpl.java

```diff
@@ -50,11 +50,10 @@ public void saveEventForm(final EventForm eventForm) {
             });
     eventForm
         .getBillings()
-        .forEach((
-            x -> {
+            .forEach(
+                    (x -> {
```

Comment:
better name for variable

Reply:
You can also use map instead (with method reference ) of foreach then at the end foreach billingFormList::add